### DISCUSSION TITLE
Open last view/context on start

### DISCUSSION THREAD

**Discussion by tomzx** [damionx7/Zettel-Notes-Documentation#79] [Feature Requests]
Currently you can enable to view the last opened note on start. In this feature request, the idea would be to open up where we last left Zettel, i.e., if we were in a note, then back in this note, if we were looking at a directory's notes, then back to this repository+directory. This makes it seamless when the app is closed.

**Comment by vupogorelov** [comment]
This convenience is very lacking.
